src/js/evented.js

```diff
@@ -21,7 +21,7 @@ export class Evented {
 
   off(event, handler) {
     if (typeof this.bindings === 'undefined' || typeof this.bindings[event] === 'undefined') {
-      return;
+      return false;
```

Comment:
Why did we change to return false here? Not saying it's wrong, just want to understand what it does for us.

Reply:
clearer response for tests

Reply:
I wonder if maybe we should skip the return part, and just wrap the whole method in 
```js
if (!_.isUndefined(this.bindings) && !_.isUndefined(this.bindings[event])) {
```

Reply:
Perhaps we should save that for a later refactor though, since we might replace evented anyway